modules/electrophysiology_browser/jsx/react-series-data-viewer/src/series/components/Form.js

```diff
@@ -384,3 +419,454 @@ TextboxElement.propTypes = {
   readonly: PropTypes.bool,
   help: PropTypes.string,
 };
+
+/**
+ * Checkbox Component
+ * React wrapper for a <input type="checkbox"> element.
+ */
+export const CheckboxElement = (props) => {
```

Comment:
Resolved by c442f0e91e2787faff9b112ae6b62b4cdd01e2a5.

I did not create the file but I suspect it was created to bypass some style constraints, as I initially did with the `CheckboxElement` I added. For the other elements I added below it, it was because I was encountering an issue with being able to see the `SelectDropdown` component in `jsx/MultiSelectDropdown.js`. Turns out the there are some missing `export` keywords in  `jsx/MultiSelectDropdown.js` and I added them in the commit stated above.

I made some style changes to be able to remove the redundant components that I added. It now uses the existing Loris components.